start.yaml

```diff
@@ -18,6 +18,7 @@ max health: 15
 x: 0
 xp: 0
 y: 0
+map_zone: "Kathallion Plains"
```

Comment:
You're in a dungeon

Reply:
But before entering the dungeon, you're outside right?

Reply:
Yes, but you enter the dungeon before you start playing

Reply:
Also, I've been using spaces in my data file variables:
```suggestion
map zone: "Dungeon Entrance"
```

Reply:
I'll fix that later.

Reply:
I fixed that by commit aa666c7.